static/js/constants.js

```diff
@@ -96,15 +96,18 @@ export const EMPLOYMENT_LEVEL = [
 ]
 
 export const HIGHEST_EDUCATION_CHOICES = [
-  [1, "Doctorate"],
-  [2, "Master's or professional degree"],
-  [3, "Bachelor's degree"],
-  [4, "Associate degree"],
-  [5, "Secondary/high school"],
-  [6, "Junior secondary/junior high/middle school"],
-  [7, "Elementary/primary school"],
-  [8, "No formal education"],
-  [0, "Other education"]
+  ["Doctorate", "Doctorate"],
+  ["Master's or professional degree", "Master's or professional degree"],
+  ["Bachelor's degree", "Bachelor's degree"],
+  ["Associate degree", "Associate degree"],
+  ["Secondary/high school", "Secondary/high school"],
+  [
+    "Junior secondary/junior high/middle school",
+    "Junior secondary/junior high/middle school"
+  ],
+  ["Elementary/primary school", "Elementary/primary school"],
+  ["No formal education", "No formal education"],
+  ["Other education", "Other education"]
```

Comment:
You could rewrite this as an array of single values like `EMPLOYMENT_LEVEL` and `EMPLOYMENT_FUNCTION` (then change the profile edit form accordingly) since the value pairs here are always identical.